writer.go

```diff
@@ -99,27 +153,35 @@ func newWriterDictLevel(w io.Writer, cd *CDict, compressionLevel int) *Writer {
 }
 
 // Reset resets zw to write to w using the given dictionary cd and the given
-// compressionLevel.
+// compressionLevel. Other parameters set via WriterParams remain unchanged.
 func (zw *Writer) Reset(w io.Writer, cd *CDict, compressionLevel int) {
```

Comment:
I'd suggest adding `ResetWriterParams(w io.Writer, params *WriterParams)` function as well for updating arbitrary set of params.

Reply:
done